### PR TITLE
Create cdn.xrap1dx.json

### PR DESCRIPTION
**Pull request by xrap1dx**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] The file's name is all lowercased and alphanumeric. 
- [x] You have completed your website. 
- [x] The website is reachable.  
- [x] You're not using Vercel.  
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.

### CODE REVIEW

File: cdn.xrap1dx.json
Comment (CuteDog5695) [inline cdn.xrap1dx.json:1]: File needs to be in domains directory

**Review by iostpa — CHANGES_REQUESTED** [review]

File: cdn.xrap1dx.json
Comment (xrap1dx) [reply cdn.xrap1dx.json:1]: fixed

**Review by xrap1dx — COMMENTED** [review]
fixed here ^^

**Review by iostpa — APPROVED** [review]
Seems good to me now

**Pull request merged**